egs/voxceleb/README.txt

```diff
@@ -1,6 +1,13 @@
-This is a Kaldi recipe for speaker verification using the VoxCeleb1 and VoxCeleb2 corpora.
-See http://www.robots.ox.ac.uk/~vgg/data/voxceleb/ and 
-http://www.robots.ox.ac.uk/~vgg/data/voxceleb2/ for additional details and
-information on how to obtain them.
+ 
+ This is a Kaldi recipe for speaker verification using the VoxCeleb1 and
+ VoxCeleb2 corpora.  See http://www.robots.ox.ac.uk/~vgg/data/voxceleb/ and 
+ http://www.robots.ox.ac.uk/~vgg/data/voxceleb2/ for additional details and
+ information on how to obtain them.
 
-Note: This recipe requires ffmpeg to be installed and its location included in $PATH
+ Note: This recipe requires ffmpeg to be installed and its location included
```

Comment:
It would be nice if run.sh or a script in local/ would check for this, if not done already.

Reply:
Good idea--I'll add that.

Reply:
Done.